homeassistant/components/light/group.py

```diff
@@ -79,10 +79,11 @@ def async_state_changed_listener(entity_id: str, old_state: State,
 
         self._async_unsub_state_changed = async_track_state_change(
             self.hass, self._entity_ids, async_state_changed_listener)
+        self.async_schedule_update_ha_state(True)
```

Comment:
You should just call `await self.async_update()` here, updating state is done right after the call to `async_added_to_hass`: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity_platform.py#L257-L260

Reply:
Good catch, thanks.